MediaRecorder.html

```diff
@@ -775,97 +775,101 @@
+        };
+      </pre>
+      <section>
+        <h2>Constructors</h2>
+        <dl data-link-for="MediaRecorderErrorEvent"
+            data-dfn-for="MediaRecorderErrorEvent" class="constructors">
+        <dt><code>MediaRecorderErrorEvent</code></dt>
+        <dd>
+          Constructs a new <code><a>MediaRecorderErrorEvent</a></code>.
+          <table class="parameters">
+          <tbody>
+            <tr>
+              <th>Parameter</th>
+              <th>Type</th>
+              <th>Nullable</th>
+              <th>Optional</th>
+              <th>Description</th>
+            </tr>
+            <tr>
+              <td class="prmName">type</td>
+              <td class="prmType"><code>DOMString</code></td>
+              <td class="prmNullFalse"><span role="img" aria-label="False">&#10008;</span></td>
+              <td class="prmOptFalse"><span role="img" aria-label="False">&#10008;</span></td>
+              <td class="prmDesc">
+              The
+              <a href="https://heycam.github.io/webidl/#idl-DOMException-error-names">
+              error name</a> to be used in constructing
+              <a data-link-for="MediaRecorderErrorEvent">error</a>.
+              </td>
+            </tr>
+            <tr>
+              <td class="prmName">eventInitDict</td>
+              <td class="prmType"><a class="idlType"><code>MediaRecorderErrorEventInit</code></a></td>
+              <td class="prmNullFalse"><span role="img" aria-label="False">&#10008;</span></td>
+              <td class="prmOptTrue"><span role="img" aria-label="False">&#10004;</span></td>
+              <td class="prmDesc"></td>
+            </tr>
+          </tbody>
+          </table>
+        </dd>
+        </dl>
+      </section>
+      <section>
+        <h2>Attributes</h2>
+        <dl data-link-for="MediaRecorderErrorEvent"
+            data-dfn-for="MediaRecorderErrorEvent" class="attributes">
+        <dt><dfn><code>error</code></dfn> of type <span class="idlAttrType">
+          <a href="https://heycam.github.io/webidl/#dfn-DOMException">
+          DOMException</a></span>, readonly</dt>
+        <dd>The DOMException error that triggered the event, if any.</dd>
+        </dl>
+      </section>
+      <section>
+        <h2>Dictionary <a class="idlType">MediaRecorderErrorEventInit</a> members</h2>
+        <dl data-link-for="MediaRecorderErrorEventInit"
+            data-dfn-for="MediaRecorderErrorEventInit" class="dictionary-members">
+        <dt><dfn><code>error</code></dfn> of type <span class="idlAttrType">
+          <a href="https://heycam.github.io/webidl/#dfn-DOMException">
+          DOMException</a></span>, readonly</dt>
+        <dd>The <a href="https://heycam.github.io/webidl/#dfn-DOMException">
+          DOMException</a> causing the error that triggered the event.
+          An explanatory message about the error circumstances MAY be provided
+          in its <a href="https://www.w3.org/TR/WebIDL-1/#es-DOMException-constructor-object">
```

Comment:
https://heycam.github.io/webidl/#dfn-DOMException please; it has several important fixes